### PR TITLE
Don't prevent Lexer errors from executing mongo scrapbook queries

### PR DESCRIPTION
**Pull request by nturinski**
Fixes https://github.com/microsoft/vscode-cosmosdb/issues/1193, https://github.com/microsoft/vscode-cosmosdb/issues/981

Today the lexer is throwing an error for valid mongo commands.  Updating the mongo.g4 file (where the lexer is pulling the grammar) is not a trivial task that I think will take me a lot of investigating to do.

However, if we don't block users from executing commands that the lexer considers invalid, users can still workaround it.

I don't think this is quite a proper fix, perse, but at least it will unblock the users who have run into the above errors.

It looks like this if a mongo command _actually_ fails:
image

### CODE REVIEW

File: src/mongo/MongoScrapbook.ts [hunk 1]
```diff
@@ -109,7 +109,8 @@ async function executeCommand(activeEditor: vscode.TextEditor, database: MongoDa
 		if (command.errors && command.errors.length > 0) {
 			//Currently, we take the first error pushed. Tests correlate that the parser visits errors in left-to-right, top-to-bottom.
 			const err = command.errors[0];
-			throw new Error(`Error near line ${err.range.start.line}, column ${err.range.start.character}: '${err.message}'. Please check syntax.`);
+			// don't await intentionally
+			ext.ui.showWarningMessage(`Error near line ${err.range.start.line}, column ${err.range.start.character}: '${err.message}'. Please check syntax.`);
```
Comment (EricJizbaMSFT) [inline src/mongo/MongoScrapbook.ts]: You said it's not trivial to update the mongo.g4 file, but this change doesn't feel trivial either. It feels like a pretty significant behavior change that affects many code paths and we would have to spend a decent amount of time ensuring it doesn't have negative effects. IMO it seems more valuable to spend the time on the mongo.g4 file and properly fix the main issue

A few reasons I don't like this:
1. We no longer get these errors in telemetry
1. If the error here is a false positive, it feels confusing to show an error when the command will actually succeed
1. If the error is accurate, users will get duplicate errors (one as a message, one in the file)
1. If we have syntax errors - that means we can't parse something. Sounds risky to execute a command we can't even parse (what if it sends half of one command and half of another)

Comment (StephenWeatherford) [reply src/mongo/MongoScrapbook.ts]: Agreed, you should fix the mongo.g4 file, this change would have major effects down the road.  I've got some experience with it, let me know if you'd like help.

Comment (nturinski) [reply src/mongo/MongoScrapbook.ts]: Please and thank you!

**Comment by nturinski** [comment]
Closing due to undesirable effects.

**Pull request closed without merging**